Game statistics

m-mrcr opened the pull request.

m-mrcr committed:
added note from conversation with megan

m-mrcr committed:
changed indentation

m-mrcr committed:
added comment from conversation with megan

m-mrcr committed:
coverage stuff

m-mrcr committed:
commented out what we've used so far

m-mrcr committed:
added least_popular_venue method

m-mrcr committed:
added percentage home wins test and implementation

m-mrcr committed:
this adds final tests and implementation to iteration 2

n-flint commented:
Hey! Looks great!, Maybe we could create the instance of stattracker.new in the setup method for the tests, instead of calling it for every test. What do you guys think?

juliamazey committed:
Rearranged rows so that they are sorted by season

juliamazey committed:
Changed set up

  Hey guys y added the StatTracker.from_csv method to the setup as an instance variable @stat_tracker so we don't have to initialize on every test. I think this should work

juliamazey reviewed (changes requested):
Hey guys, I changes some things. What do you think?

m-mrcr reviewed (commented):
This is great. I was staring at this forever yesterday, and it'll just be better to have a sample sorted with some kind of order.

m-mrcr reviewed (commented):
This seems like it makes sense? I'd be fine committing it and reverting if we need to, but I like that we could delete some code by making this an i-var.

n-flint reviewed (approved):
This looks great Julia!

n-flint reviewed (approved):
This also looks good, thanks Julia!

n-flint reviewed (commented):
Hey there are a number of helper methods we don't have a test for. Ill make a list of the ones that we need a test for.

n-flint committed:
Update game_statistics.rb

n-flint reviewed (commented):
Hey, I also added a few comments about adding tests to the helper methods.
